src/bundle/Resources/views/themes/admin/ui/modal/delete_confirmation.html.twig

```diff
@@ -0,0 +1,35 @@
+{% set confirm_button_rendered %}
```

Comment:
Do we need these twig variables? Why not just put blocks in the place of rendering? 🤔

Reply:
Unfortunately we need them, as we can't put its content inside embed (in line 33/34) as in such case we won't be able to embed `delete_confirmation.html.twig` and overwrite `confirm_button`/`cancel_button`

Reply:
Right, I forgot about `embed` :)